netebpfext/net_ebpf_ext_bind.c

```diff
@@ -307,15 +307,17 @@ net_ebpf_ext_resource_allocation_classify(
         goto Exit;
     }
 
-    attached_client = (net_ebpf_extension_hook_client_t*)filter_context->client_context;
-    if (attached_client == NULL) {
+    if (filter_context->client_detached) {
+        NET_EBPF_EXT_LOG_MESSAGE_NTSTATUS(
+            NET_EBPF_EXT_TRACELOG_LEVEL_VERBOSE,
+            NET_EBPF_EXT_TRACELOG_KEYWORD_SOCK_ADDR,
```

Comment:
Should this keyword be something related to bind? (Applies to below traces too)

Reply:
Also - What is the intent of this trace? It looks like we are treating this as an error case, as opposed to saying that the client has just detatched. 

Perhaps a stronger trace statement to clearly indicate a failure, such as 'net_ebpf_ext_resource_allocation_classify exiting as client has already detatched' (or similar).

Reply:
The intent here was to just log the event (primarily for subsequent debugging if needed).

Genuine 'WFP noob' question: Why would this be considered an error? Shouldn't clients detaching on their own accord be a 'normal' occurrence? Not sure what I'm missing here.

Reply:
Following up on the offline sync-up, not an error.  Added caller function in trace message, made trace message more specific.